Project.toml

```
@@ -12,9 +12,9 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 
 [compat]
-ADNLPModels = "^0.3, 0.4"
+ADNLPModels = "^0.3, 0.4, 0.5, 0.6"
```

```suggestion
ADNLPModels = "^0.3, 0.4, 0.7"
```
@dpo this works locally for me